### DISCUSSION TITLE
Customizing grafana dashboard timezones

### DISCUSSION THREAD

**Discussion by bip7686** [prometheus-operator/kube-prometheus#1284] [Q&A] [answered]
I saw that this was recently added to the kubernetes-mixin: https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/config.libsonnet#L79

But how can I configure this setting from my jsonnet file in my kube-prometheus project?

I tried something like this:
```
prometheusOperator+: {
      mixin+: {
          _config+: {
            grafanaK8s+: {
              grafanaTimezone: 'browser',
            },
          },
        },
        },
```
but it doesn't work, but I think its maybe going in the right direction. Help is really appreciated :)

**Comment by ArthurSens** [comment, marked as answer]
👋 

Kubernetes mixin is imported by the [kubernetesControlPlane](https://github.com/prometheus-operator/kube-prometheus/blob/fc136a84f61934e2224bc0487db37cb404268517/jsonnet/kube-prometheus/components/k8s-control-plane.libsonnet#L32) component. So my bet to make this work would be something like this:

```
values+::{
  kubernetesControlPlane+: {
  mixin+: {
          _config+: {
            grafanaK8s+: {
              grafanaTimezone: 'browser',
            },
          },
        },
  }
}
```

**Reply by bip7686** [reply]
Yes now it works, thank you! 😃
